packages/web/src/components/common/select-tab/SelectTab.styles.ts

```diff
@@ -23,3 +23,18 @@ export const SelectButton = styled.button`
     color: ${({ theme }) => theme.color.text11};
   }
 `;
+
+export const ChartSelectButton = styled.button`
+  ${mixins.flexbox("row", "center", "center")};
+  ${fonts.body10};
+  width: 60px;
+  height: 37px;
+  padding: 0px 8px;
+  border-radius: 4px, 0px, 0px, 4px;
```

Comment:
border-radius doesn't seem to be applied.
`border-radius: 4px 0px 0px 4px;` seems to be correct.
Also, it looks like you intended for only the border-radius of the left part to be applied, I'll have to check if the same border-radius should be applied for all items.
(My guess is that the radius of the first item would be applied to the left, and the last item to the right).

Reply:
thank you Let's check again.